.devcontainer/post-create.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+
+function add_config_if_not_exist {
+    if ! grep -F -q "$1" $HOME/.bashrc; then
+        echo "$1" >> $HOME/.bashrc
+    fi
+}
+
+add_config_if_not_exist "source /opt/ros/humble/setup.bash"
+LOCAL_SETUP_FILE=`pwd`/install/setup.bash
+add_config_if_not_exist "if [ -r $LOCAL_SETUP_FILE ]; then source $LOCAL_SETUP_FILE; fi"
+
+source /opt/ros/humble/setup.bash
+colcon build --symlink-install --continue-on-error
```

Comment:
use `--continue-on-error` as this is really only to try to build the workspace so that packages that already compile are ready and don't have to be compile later, but we expect that many will not compile as after all it's a devcontainer ;-)